babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -474,6 +474,8 @@ SqlCreate SqlCreateTable(Span s, boolean replace) :
     final SqlNodeList columnList;
     final SqlNode query;
     WithDataType withData = WithDataType.UNSPECIFIED;
+    SqlPrimaryIndex primaryIndex = null;
+    SqlIndex index;
```

Comment:
nit: Can this be marked as final?

Reply:
Actually, I realized it can't, because if multiple PRIMARY INDEX statements are declared, the way we defined behavior is that each statement will overwrite the previous one (in order to ensure that there is only ever one primary index). So for example:
CREATE TABLE FOO PRIMARY INDEX (fname) PRIMARY INDEX (lname)
is legal (but only the last one will persist).

Reply:
Ah you're right. I missed that this was in an ()* clause.